src/decode-vxlan.c

```diff
@@ -47,10 +47,16 @@
 #define VXLAN_DEFAULT_PORT      4789
 #define VXLAN_DEFAULT_PORT_S    "4789"
 
+enum TunnelPacketMode {
```

Comment:
would move this into decode.h, but maybe only after adding support for other protocols like geneve, erspan, etc